track_and_trace/asset_client/src/views/list_organizations.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const m = require('mithril')
+const sortBy = require('lodash/sortBy')
+const truncate = require('lodash/truncate')
+const {Table, FilterGroup, PagingButtons} = require('../components/tables.js')
+const api = require('../services/api')
+const organizations = require('../services/organizations')
+
+const PAGE_SIZE = 50
+
+const OrganizationList = {
```

Comment:
There's a lot of new things in the PR, that perhaps could be broken into more commits.

Reply:
Also, more detail in the commit messages would be a good change.